schema.json

```diff
@@ -82,25 +144,25 @@
         "metadata": {
           "$ref": "#/$defs/metadata"
         },
+        "stem": {
```

Comment:
Can we keep the order the same for `stem` and `type` in the `command` object. It adds noise to the PR and I just want to focus on what we are actually changing.

Reply:
Good catch; cleaned up the diff